### ISSUE TITLE
NVDA isn't announcing selection in Dev-C++

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#8946]
### Steps to reproduce:
1. Open Dev-C++.
2. Create empty file and type anything.
3. Start selecting with shift and arrows or with any other selection command.
### Actual behavior:
Nothing is announced, but text is selected.
### Expected behavior:
The selection should be announced like in other programs.
#### NVDA Installed/portable/running from source:
installed

#### NVDA version:
version alpha-16247,fbae6b0b

#### Windows version:
Windows 7 x64
#### Name and version of other software in use when reproducing the issue:
Dev-C++ version 5.11

#### Other information about your system:
With JAWS the selection isn't announced either, but by setting custom highlight it can be worked around.
The developer info for the editor control is as follows:
Developer info for navigator object:
name: None
role: ROLE_EDITABLETEXT
states: STATE_FOCUSABLE, STATE_FOCUSED
isFocusable: True
hasFocus: True
Python object: <baseObject.Dynamic_IAccessibleDisplayModelEditableTextWindowNVDAObject object at 0x036B6A90>
Python class mro: (<class 'baseObject.Dynamic_IAccessibleDisplayModelEditableTextWindowNVDAObject'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.DisplayModelEditableText'>, <class 'NVDAObjects.behaviors.EditableText'>, <class 'editableText.EditableText'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: RectLTWH(left=168, top=124, width=1194, height=550)
value: None
appModule: <'appModuleHandler' (appName u'devcpp', process ID 5124) at address 5bf35b0>
appModule.productName: u'Dev-C++'
appModule.productVersion: u'5'
TextInfo: <class 'displayModel.EditableTextDisplayModelTextInfo'>
windowHandle: 1245636
windowClassName: u'TSynEdit'
windowControlID: 1245636
windowStyle: 1442906112
windowThreadID: 6096
windowText: u''
displayText: u''
IAccessibleObject: <POINTER(IAccessible) ptr=0x6adde0 at 58e98a0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=1245636, objectID=-4, childID=0
IAccessible accName: None
IAccessible accRole: ROLE_SYSTEM_CLIENT
IAccessible accState: STATE_SYSTEM_FOCUSED, STATE_SYSTEM_FOCUSABLE, STATE_SYSTEM_VALID (1048580)
IAccessible accDescription: None
IAccessible accValue: None

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?

No

**Comment by Adriani90** [comment]
I am closing this issue as abandoned. Dev-C++ is discontinued, and has not been updated since 2015 or so.
See also this article:
https://cplusplus.com/articles/36vU7k9E/
and
https://filehippo.com/download_bloodshed-dev-c/

You might want to test rather this one:
https://github.com/skylee03/My-Dev-Cpp?tab=readme-ov-file
